source/class/qx/ui/window/Manager.js

```diff
@@ -81,6 +81,9 @@ qx.Class.define("qx.ui.window.Manager", {
      * {@link qx.ui.core.queue.Widget widget queue}.
      */
     syncWidget() {
+      if (!this.__desktop){
```

Comment:
A much more typical way of handling this type of issue in qooxdoo, is:

```
      if (this.isDisposed()) {
        return;
      }
```

Reply:
It looks much better! Tested it, does work.